scripts/generate-tfvars.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
```

Comment:
Why do we need these new files. I do not want to make major changes now. Just fixing bugs is awesome

Reply:
@swathiswaminathan what is the purpose of adding this and the other files?  How are they used in this demo?